WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -207,7 +208,7 @@ public JsonRpcResult GetStatus()
 			},
 			["onionService"] = Global.OnionServiceUri?.ToString() ?? "Unavailable",
 			["backendStatus"] = sync.BackendStatus == BackendStatus.Connected ? "Connected" : "Disconnected",
-			["bestBlockchainHeight"] = smartHeaderChain.TipHeight.ToString(),
+			["bestBlockchainHeight"] = smartHeaderChain.TipHeight.ToString(CultureInfo.InvariantCulture),
```

Comment:
This change is only to avoid a warning but I agree with you

Reply:
This change is required to fix a warning. I've added the change you suggested, however I think that the only way to avoid any mistake is to delete all `ToString()` from this class and instead rely exclusively on the `JsonConverter`